lib/engine/round/operating.rb

```diff
@@ -300,12 +311,24 @@ def process_lay_tile(action)
           @teleported = ability[:hexes].include?(hex_id) &&
           ability[:tiles].include?(action.tile.name)
         end
+
+        new_tile = action.hex.tile
```

Comment:
this is confusing, just do action.tile

Reply:
also i don't think you have to dig into the tile, hex has paths as well

Reply:
but doesn't have cities, but i've fixed the first comment.

Reply:
what do you mean?

Reply:
i'd need to dig into the tile anyway since i need to check if there's more than one city.